data/json/items/comestibles.json

```diff
@@ -992,8 +992,10 @@
         "charges" : 7,
         "phase" : "liquid",
         "comestible_type" : "DRINK",
-        "flags" : ["BREW"],
-        "brew_time" : 7200
+        "brewable" : {
+            "time" : 7200,
+            "result" : "whiskey"
```

Comment:
Yeah, this is just a refactor in preparation to fix that.

The problem is, the current id substring hack prevents adding a new step to brewing. Mash should ferment to wash, then be distilled into whiskey, but currently it can only ferment into something with its own id minus `brew_`.